src/genezio.ts

```diff
@@ -125,7 +125,11 @@ program.hook("postAction", async () => {
         });
     }
 
-    process.stdin.unref();
+    try {
+        process.stdin.unref();
```

Comment:
I think this errors because the process on remote is daemonized and doesn't have a stdin attached.

You can also add `?` to call this unref only if it's defined and a callable:
```suggestion
        process.stdin.unref?.();
```
This expands to:
```
if (process.stdin && typeof process.stdin.unref === "function") {
  process.stdin.unref();
}
```